app/controllers/application_controller.rb

```diff
@@ -637,4 +638,17 @@ def determine_custom_metadata_keys
     end
     keys
   end
+
+  def displaying_single_page?
+    instance_variable_get("@single_page")||false
```

Comment:
my comment on the call, related to this. Why not just `@single_page || false` - I don't think instance_variable_get is needed